central/plugins/org.jboss.tools.central/src/org/jboss/tools/central/editors/JBossCentralEditor.java

```diff
@@ -197,13 +198,18 @@ protected void addPages() {
 			}
 			setPageImage(index, gettingStartedImage);
 
-			softwarePage = new SoftwarePage(this);
-			index = addPage(softwarePage);
-			if (softwareImage == null) {
-				softwareImage = JBossCentralActivator.getImageDescriptor(
-						"/icons/software.png").createImage();
+			String applicationName = JBossToolsUsageActivator.getDefault().getJBossToolsEclipseEnvironment().getEclipseUserAgent().getApplicationName();
+			if (!applicationName.contains("|Devstudio")) { // Installed via RPM. Do not add "Software/Update" tab to Central. See https://issues.jboss.org/browse/JBDS-4195
```

Comment:
@nickboldt can JBT be installed via packages or is it restricted to Devstudio only?